storage/imxrt-flash/flashsrv.c

```diff
@@ -58,6 +58,8 @@ typedef struct {
 	uint8_t fID;
 	uint8_t pStatus;
 
+	uint8_t authProcPid;
```

Comment:
needs to be `pid_t` - otherwise I can use uint8_t overflow to impersonate the other process

Reply:
done